.gitignore

```diff
@@ -1,2 +1 @@
-
-build*
+build
```

Comment:
I don't understand this change. I frequently create directories like build-xcode and build-afl and I don't want any of them to be tracked by git.

Reply:
Ok, but this also means that any file that starts with "build" or "Build" can not be added here.
Maybe "build*/" would work here? If not, just reject the pull request.

Reply:
> Ok, but this also means that any file that starts with "build" or "Build" can not be added here.

Correct; that is the intention